TransparentCalculator/TransparentCalculator/MainWindow.xaml

```diff
@@ -5,7 +5,7 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:TransparentCalculator"
         mc:Ignorable="d"
-        Title="MainWindow" Height="300" Width="250">
+        Title="MainWindow" Height="300" Width="250" AllowsTransparency="True" WindowStyle="None" MouseDown="Window_MouseDown">
```

Comment:
Ja. Hvis man slår AllowsTransparency til skal WindowStyle være ”None”.

Fra: dvalinor [mailto:notifications@github.com]
Sendt: 17. august 2016 10:57
Til: dvalinor/TransparentCalculator <TransparentCalculator@noreply.github.com>
Cc: Troels Bastrup Mikkelsen <troe2306@edu.campusvejle.dk>; Author <author@noreply.github.com>
Emne: Re: [dvalinor/TransparentCalculator] Added input handler and made window draggable (#2)


In TransparentCalculator/TransparentCalculator/MainWindow.xaml<https://github.com/dvalinor/TransparentCalculator/pull/2#discussion_r75085714>:

> @@ -5,7 +5,7 @@

>          xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"

>          xmlns:local="clr-namespace:TransparentCalculator"

>          mc:Ignorable="d"

> -        Title="MainWindow" Height="300" Width="250">

> +        Title="MainWindow" Height="300" Width="250" AllowsTransparency="True" WindowStyle="None" MouseDown="Window_MouseDown">

Er det nødvendigt at vi har WindowStyle="None"?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/dvalinor/TransparentCalculator/pull/2/files/b6df0dc3324a94fbaf87af0455ccc66c7432a700#r75085714>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AUFHgMsPoU5_GU1KQy7BdTHqctxGInuXks5qgszQgaJpZM4JmOq8>.